packages/aws-client/src/StepFunctions.ts

```diff
@@ -53,11 +63,9 @@ export const describeExecution = improveStackTrace(
  *
```

Comment:
Suggestion - fix jsdoc or remove param specifiers (can't recall what our current agreement is re: this)